CoreFoundation/NumberDate.subproj/CFDate.c

```diff
@@ -40,6 +40,7 @@
 
 const CFTimeInterval kCFAbsoluteTimeIntervalSince1970 = 978307200.0L;
 const CFTimeInterval kCFAbsoluteTimeIntervalSince1904 = 3061152000.0L;
+const CFTimeInterval kCFAbsoluteTimeIntervalSince1601 = 12622780800.0L;
```

Comment:
This needs to be `static`.

Reply:
Actually, shouldn't all three of those be `static`?

Reply:
The other two are API.

Reply:
https://developer.apple.com/documentation/corefoundation/kcfabsolutetimeintervalsince1970
https://developer.apple.com/documentation/corefoundation/kcfabsolutetimeintervalsince1904